### PR TITLE
build: add build numbers timestamps only for debug builds

### PR DESCRIPTION
**Pull request by ranj063**
Signed-off-by: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>

### CODE REVIEW

**Comment by cujomalainey** [comment]
Would it be worth putting the shortened git hash in the image as well for debug images?

**Comment by ranj063** [comment]
@cujomalainey the git hash is included in the image too. SOF_TAG takes care of it.

**Comment by cujomalainey** [comment]
Got it! Thanks

File: scripts/xtensa-build-all.sh [hunk 1]
```diff
@@ -177,7 +177,7 @@ do
 		PATH=$pwd/../$HOST/bin:$OLDPATH
 	fi
 
-	./configure --with-arch=xtensa --with-platform=$PLATFORM --with-root-dir=$ROOT --host=$HOST \
+	./configure --with-arch=xtensa --with-platform=$PLATFORM --with-root-dir=$ROOT --host=$HOST --enable-debug=no\
```
Comment (lgirdwood) [inline scripts/xtensa-build-all.sh]: Can we pass this into xtensa-build-all.sh as a -d switch.

File: src/platform/baytrail/platform.c [hunk 2]
```diff
@@ -65,11 +65,14 @@ static const struct sof_ipc_fw_ready ready
 	},
 	/* dspbox is for DSP initiated IPC, hostbox is for host initiated IPC */
 	.version = {
+	/* Add build number and timestamp for debug build */
+#ifdef DEBUG_BUILD
 		.build = SOF_BUILD,
```
Comment (lgirdwood) [inline src/platform/baytrail/platform.c]: Version numbers should be included by default.

**Comment by lgirdwood** [comment]
@ranj063 please see https://github.com/thesofproject/sof/pull/529

File: scripts/xtensa-build-all.sh (hunk 1 above)
Comment (xiulipan) [reply scripts/xtensa-build-all.sh]: Thas is a good idea

File: src/platform/baytrail/platform.c [hunk 3]
```diff
@@ -65,11 +65,14 @@ static const struct sof_ipc_fw_ready ready
...
 		.minor = SOF_MINOR,
```
Comment (xiulipan) [inline src/platform/baytrail/platform.c]: Are you sure about remove SOF version?
which means 1.0 or 1.1 is not needed?
I think you only need to remove date, time and build

**Comment by ranj063** [comment]
closing this as #529 will take care of this.

**Pull request closed without merging**